pkg/git/git_test.go

```diff
@@ -80,6 +86,14 @@ var _ = Describe("GitClient", func() {
 
 		Expect(fakeExecutor.Calls[1].Arguments[0]).To(Equal("git"))
 		Expect(fakeExecutor.Calls[1].Arguments[2]).To(
+			Equal([]string{"config", "user.email", "test-user@example.com"}))
+
+		Expect(fakeExecutor.Calls[2].Arguments[0]).To(Equal("git"))
+		Expect(fakeExecutor.Calls[2].Arguments[2]).To(
+			Equal([]string{"config", "user.name", "test-user"}))
+
+		Expect(fakeExecutor.Calls[3].Arguments[0]).To(Equal("git"))
```

Comment:
I was looking at gomega docs a little more lately, and spotted this: https://godoc.org/github.com/onsi/gomega#ConsistOf

```Go
Expect(fakeExecutor.Calls[2].Arguments).To(ConsistOf("git", "<what's here?>", "commit", "-m", "test commit", "--author=test-user <test-user@example.com>"))
```

Perhaps also would be nice to make this table-driven...